src/middlewares/auth.middleware.js

```diff
@@ -1,15 +1,14 @@
 import user from "../models/user.model.js";
 import jwt from "jsonwebtoken";
 
-const verifyToken = util.promisify(jwt.verify);
 
 const validateToken = async (req, res, next) => {
   let token
-  let authHeader = req.headers.Authorization || req.headers.authorization
+  let authHeader = req.headers.Authorization || req.headers.authorization;
   if (authHeader && authHeader.startsWith("Bearer")) {
     token = authHeader.split(" ")[1]
     try {
-      const decoded = await verifyToken(token, process.env.JWT_KEY)
+      const decoded = jwt.verify(token, process.env.JWT_KEY)
```

Comment:
@DikshantK2004 is there any specific reason to remove await here?

Reply:
No such reason, just that verify is a synchronous and someone had used a promisify to convert it to asynchronous but may be they had not imported something.  My VS code was not helping with it. So, I removed it.

Reply:
@DikshantK2004 ok.